lib/cloud/metadata.rego

```diff
@@ -0,0 +1,18 @@
+# METADATA
+# custom:
+#   library: true
+package lib.cloud.metadata
+
+import rego.v1
+
+# Returns the object found by the given path
+# if child object is not found, returns the last found object
+obj_by_path(obj, path) := res if {
```

Comment:
@simar7  Maybe there is a better name for this function?

Reply:
I think it's fine, it's clear to me what the function does.